# HG changeset patch
# User Avinoam
# Date 1427054513 -7200
#      Sun Mar 22 22:01:53 2015 +0200
# Node ID 83dc1ab95429c9beef9de6bbfe50ccdacc7834d8
# Parent  5e6e0563388b26d9180bfcff2c517e4876325e02
__w2mpth__.m: fix typos

* scripts/miscellaneous/private/__w2mpth__.m: change __w2mpth to __w2mpth__

diff --git a/scripts/miscellaneous/private/__w2mpth__.m b/scripts/miscellaneous/private/__w2mpth__.m
--- a/scripts/miscellaneous/private/__w2mpth__.m
+++ b/scripts/miscellaneous/private/__w2mpth__.m
@@ -21,38 +21,38 @@
 ## Convert a Windows-style relative or full path name to MinGW style.
 ##
 ## @strong{Caution:} __w2mpth__ does not check the validity of the path.
 ##
 ## Examples:
 ##
 ## @example
 ## @group
-##   mpth = __w2mpth ('D:\full\path\to\file.dat')
+##   mpth = __w2mpth__ ('D:\full\path\to\file.dat')
 ##   @result{} '/D/full/path/to/file.dat'
 ## @end group
 ## @end example
 ##
 ## @example
 ## @group
-##   mpth = __w2mpth ('relative\path\to\file.dat')
+##   mpth = __w2mpth__ ('relative\path\to\file.dat')
 ##   @result{} 'relative/path/to/file.dat'
 ## @end group
 ## @end example
 ##
 ## @end deftypefn
 
 ## Author: Philip Nienhuis <prnienhuis@users.sf.net>
 ## Created: 2015-01-16
 
 function mingwpath = __w2mpth__ (winpath)
 
   ## Check for platform
   if (! ispc)
-    error ("__m2wpath__ should only be called on Windows platforms\n");
+    error ("__w2mpth__ should only be called on Windows platforms\n");
   endif
 
   ## Replace backslash file separators by forward slashes
   mingwpath = strrep (winpath, '\', '/');
   ## Also treat drive letter but beware of relative filenames
   mingwpath = regexprep (mingwpath, '^([a-zA-Z]):', '/$1');
 
 endfunction
